rcsbsearchapi/search.py

```diff
@@ -1023,7 +1024,8 @@ def __iter__(self) -> Iterator[str]:
             return  # be explicit for mypy
         identifiers = self._extract_identifiers(response)
         start += self.rows
-        logging.debug(f"Got {len(identifiers)} ids")
+        # logging.debug(f"Got {len(identifiers)} ids")
```

Comment:
@MoonWalker30 Can you delete these commented out "old-version" lines? (here and elsewhere, e.g. lines 1000, 1015, 1044, ...)